django_airavata/apps/api/datastore.py

```diff
@@ -50,6 +82,7 @@ def save(username, project_name, experiment_name, file):
     file_path = os.path.join(
         exp_dir,
         experiment_data_storage.get_valid_name(file_name))
+    print("The default file path is ", file_path);
```

Comment:
Please remove this print statement. If you think this is useful for debugging, change it to a `logger.debug()` statement instead.